cmd/execution_builder.go

```diff
@@ -413,6 +413,10 @@ func (exeNode *ExecutionNode) LoadBlobService(
 		opts = append(opts, blob.WithReprovideInterval(-1))
 	}
 
+	if !node.BitswapBloomCacheEnabled {
+		opts = append(opts, blob.WithSkipBloomCache(true))
+	}
```

Comment:
Maybe always call `WithSkipBloomCache(!node.BitswapBloomCacheEnabled)` to prevent flags being out of sync if `SkipBloomCache` default value is changed.

```suggestion
	opts = append(opts, blob.WithSkipBloomCache(!node.BitswapBloomCacheEnabled))
```